### DISCUSSION TITLE
drag elements in visual mode to change their order?

### DISCUSSION THREAD

**Discussion by MayaGans** [quarto-dev/quarto-cli#486] [Feature Requests]
I have no idea how hard this would be to implement but I keep finding myself wanting to be able to drag chunks to change their order so I figured I'd let you know - a girl can dream!

**Comment by jjallaire** [comment]
We will definitely do something like this when start working on the visual editor again (currently more focused on the quarto cli but will be back to the editor soon!)

**Comment by mcanouil** [comment]
Closing this here, but feel free to open it (if not already) in <https://github.com/quarto-dev/quarto>.
